app/components/donation/donation-container.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  computed: { bool }
+} = Ember;
+
+export default Component.extend({
+  classNames: ['donation-container'],
+  donationAmount: 0,
+  projectTitle: null,
+
+  canDonate: bool('projectTitle')
```

Comment:
On the component level, I don't think so. This component is meant to be used with a project that has a title.  If the title isn't present (somehow), we display some sort of message indicating that no project is selected.  In that sense, we've covered our one edge case.